nolearn/lasagne/base.py

```diff
@@ -83,6 +83,23 @@ def __getstate__(self):
         return state
 
 
+def train_split(X, y, net, eval_size=0.2):
```

Comment:
Because the goal here is to make the interface easy to grasp to reuse, and a function is easier than a class.